### PR TITLE
Add connector spec ids

### PR DESCRIPTION
**Pull request by AKoll**
For implementing saving and backend communication later

### CODE REVIEW

**Comment by AKoll** [comment]
> Index might be a better option. We could also discuss whether ids should be unique per input section and output section or for the full node.

I believe using an index would inhibit some crucial improvements like reordering of connectors or partial update communication with the back-end. I am particularly concerned about keeping variadic connectors together how they belong, which would turn out to be a breeze with spec ids, allowing for arbitrary out-of-sequence reordering and renaming (which i deem to be especially important for group connectors).

File: src/components/nodes/Number.vue [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default {
     inputs: [],
     outputs: [
       {
+        id: 'output',
```
Comment (AKoll) [inline src/components/nodes/Number.vue]: Should this be 'number' or is 'output' appropriate for all value nodes?

Comment (Pikaju) [reply src/components/nodes/Number.vue]: "value" would also be a possiblity.

Comment (AKoll) [reply src/components/nodes/Number.vue]: This applies to the connector label as well.

**Pull request merged**